docs/deploy-options.md

```diff
@@ -155,7 +191,10 @@ If you can't or don't want to use a Service of `type: LoadBalancer` there are ot
 
 If your cluster doesn't have the capability to configure a Kubernetes LoadBalancer,
 or if you want to configure the load balancer outside Kubernetes,
-you can change the Envoy Service in the  `02-service-envoy.yaml` file to set `type` to `NodePort`.
+you can change the Envoy Service in the `02-service-envoy.yaml` file and set `type` to `NodePort`.
+  * If you have cloned down the projectcontour git repo this file can be found at:
```

Comment:
Could we link back to the Envoy service file in the location you specified and remove the phrase about "if you have cloned down the..."?

Reply:
ya no problem will update